src/storage/slotted_page.c

```diff
@@ -1266,6 +1266,9 @@ spage_compact (THREAD_ENTRY * thread_p, PAGE_PTR page_p)
   page_header_p->offset_to_free_area = to_offset;
   ASSERT_ALIGN ((char *) page_p + page_header_p->offset_to_free_area, page_header_p->alignment);
 
+  // zero-out the remaining free space
+  memset ((void *) page_p + page_header_p->offset_to_free_area, 0, page_header_p->total_free);
```

Comment:
Nit. the same value, but the meaning seems more correct.
```suggestion
  memset ((void *) page_p + page_header_p->offset_to_free_area, 0, page_header_p->cont_free);
```

Reply:
Agree. Done.